src/templates/resources/footer.json

```diff
@@ -2544,50 +2544,50 @@
         "cuisines": "[Ţöþẋ çûîšîñéšẋẋẋ]",
         "cuisineLinks": [
             {
-                "url": "/chinese-takeaways",
+                "url": "/takeaway/nearme/chinese",
                 "text": "[Çĥîñéšéẋẋẋ]"
             },
             {
-                "url": "/indian-takeaways",
+                "url": "/takeaway/nearme/indian",
                 "text": "[Îñðîåñẋẋ]"
             },
             {
-                "url": "/italian-takeaways",
+                "url": "/takeaway/nearme/italian",
                 "text": "[Îţåļîåñẋẋẋ]"
             },
             {
-                "url": "/sushi-takeaways",
+                "url": "/takeaway/nearme/sushi",
```

Comment:
I thing here it should be `/takeaway/nearme/japanese` too

Reply:
Just double-checked this with Gerry and you're right, I'll update now! 🐟